### PR TITLE
[FIX] Reduced the amount of RUN run lines to avoid the limit to 127 issue

### PR DESCRIPTION
**Pull request by ruiztulio**
All apt-get related are together now so the pip and localization ones.

**NOTE**: Merge and build this before vauxoo/docker-odoo-image#7

### CODE REVIEW

File: Dockerfile [hunk 1]
```diff
@@ -1,19 +1,19 @@
...
-RUN ln -s /usr/share/i18n/SUPPORTED /var/lib/locales/supported.d/all \
-    && locale-gen
+    && update-locale LC_ALL=en_US.UTF-8 \
+    && ln -s /usr/share/i18n/SUPPORTED /var/lib/locales/supported.d/all \
+    && locale-gen \
+    && echo 'LANG="en_US.UTF-8"' > /etc/default/locale
 ENV PYTHONIOENCODING utf-8
 ENV LANG C.UTF-8
-RUN echo 'LANG="en_US.UTF-8"' > /etc/default/locale
 ENV TERM xterm
```
Comment (hugho-ad) [inline Dockerfile]: @ruiztulio 

Is possible put Line 13 and 12  with line 11?

```docker
ENV PYTHONIOENCODING utf-8 \ 
LANG C.UTF-8 \ 
TERM xterm
```

File: Dockerfile [hunk 2]
```diff
@@ -38,7 +41,11 @@ RUN apt-get update -q && apt-get upgrade -q \
     libffi-dev \
     libssl-dev \
     vim-nox
```
Comment (moylop260) [inline Dockerfile]: Could you add a `rm` after a `update` like as https://github.com/nginxinc/docker-nginx/blob/aecb18605af6a33c1968c51df44e01e5f5a821e6/Dockerfile#L11?

Comment (ruiztulio) [inline Dockerfile]: @moylop260 as I told you in the other pr I do a single clean up at [the end](https://github.com/vauxoo-dev/docker-ubuntu-base/blob/master-reduce_runs-truiz/Dockerfile#L49)

**Comment by moylop260** [comment]
:+1:

**Pull request merged**